### PR TITLE
gcc-wrapper: point shebang to python3

### PR DESCRIPTION
**Pull request by m0veax**
The python command needs to point to python3 to build.

There are several ways to archive that. Example:

```
sudo apt install python-is-python3
```

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 We are working on a vendor Kernel for the TP-Link M7350 V3 here. We grabed the Sources from the TP-Link GPL Section and started patching to get it building with a modern Toolchain.
 
+To build this kernel, the python command needs to point to a python3 environment.
```
Comment (orangecms) [inline README]: ... the `python` command needs to be Python 3

just that; "environment" may be confusing

**Comment by m0veax** [comment]
As discussed in our direct conversation, I‘ll add python3 to the shebang and will try to compile the kernel with my python command pointing to python2.

This should be simpler as a reference in a README no one reads really before running in an error.

**Comment by orangecms** [comment]
I've set this PR to a draft in the meantime, sounds good!

**Review by orangecms — APPROVED** [review]
A single byte for us, a kernel build for mankind.

**Pull request merged**